cli/connect/console.go

```diff
@@ -199,6 +192,9 @@ func getExecutor(console *Console) prompt.Executor {
 				log.Debug(err.Error())
 			}
 		}
+		if err := console.prompt.PushToHistory(strings.TrimSpace(console.input)); err != nil {
```

Comment:
Can we avoid double trimming of the input and use the same trimmed string for the above pushing and this one?

Reply:
Of course.